### PR TITLE
fix(logs): Update PLR logs fetch logic to match the new tekton results changes

### PR DESCRIPTION
**Pull request by karthikjeeyar**
## Fixes 

https://issues.redhat.com/browse/RHTAPBUGS-1093

## Description

PipelineRun logs is not streaming properly in the UI because the tekton results logs API is now returning raw logs content directly, so UI doesn't need to decode the value again.

tekton-results PR introduced this change - https://github.com/tektoncd/results/pull/632

## Type of change

- [x] Bugfix

## Screen shots / Gifs for design review 

Before:

image

After:

image

## How to test or reproduce?

- Visit logs tab in the pipelinerun details page

## Browser conformance: 

- [x] Chrome
- [x] Firefox
- [x] Safari
- [ ] Edge

### CODE REVIEW

File: src/utils/tekton-results.ts [hunk 1]
```diff
@@ -329,10 +330,11 @@ export const getTaskRuns = (
   cacheKey?: string,
 ) => getFilteredTaskRuns(workspace, namespace, '', options, nextPageToken, cacheKey);
 
-const getLog = (workspace: string, taskRunPath: string) =>
-  commonFetchJSON<Log>(
+const getLog = async (workspace: string, taskRunPath: string) => {
```
Comment (jeff-phillips-18) [inline src/utils/tekton-results.ts:333]: nit: No reason to make this async

Comment (karthikjeeyar) [reply src/utils/tekton-results.ts]: updated

**Pull request merged**